src/devno.rs

```diff
@@ -9,6 +9,14 @@ use crate::err::Result;
 /// Device number
 pub struct BlkidDevno(libc::dev_t);
 
+/// Type alias for device major number
+#[allow(non_camel_case_types)]
+pub type maj_t = libc::dev_t;
+
+/// Type alias for device minor number
+#[allow(non_camel_case_types)]
+pub type min_t = libc::dev_t;
```

Comment:
I think maybe it would be best to conditionally compile the typedefs as their actual types in the C type signature. That avoids needing to conditionally compile the methods themselves while also making the type definition more correct.

Reply:
I agree. I'll need to use the real types as defined in the C header because now the build is broken. Looks like dev_t is 64-bit on Linux.